pre_run.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+import datetime
+from dateutil.relativedelta import relativedelta
+
+
+default_settings = {
+    "included_ticker_lists": ["dividend_aristocrats", "dividaat_list"],
+    "start_date": datetime.datetime.strftime(datetime.datetime.now() - relativedelta(years=10), '%Y-%m-%d'),
+    "end_date": datetime.datetime.strftime(datetime.date.today(), '%Y-%m-%d')
+}
+        
+class Settings():
+    def __init__(self) -> None:
+        self.user_file_name = './settings.json'
+        self.settings = dict
```

Comment:
use self.settings = {}
self._included_ticker_list = []

Reply:
I prefer to use the dict() and list() notation as it is better compatible with the other variables here.